src/convertToObject.js

```diff
@@ -3,16 +3,30 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
- * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
- *
+
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const styleObject = {};
+  const sourceArr = sourceString
+    .split(';\n')
```

Comment:
The filter operation might not be necessary. If the goal is to avoid empty strings, a better approach might be to use trim on the original string before splitting it.

Reply:
trim cannot delete all empty lines from the original string